DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Operations/Send/IRootObjectSender.cs

```diff
@@ -0,0 +1,22 @@
+using Speckle.Core.Models;
+
+namespace Speckle.Connectors.ArcGis.Operations.Send;
+
+//poc: duped from rhino
```

Comment:
If this is copy/pasted from Rhino as-is, then clearly it should live in a common project. @BovineOx any suggestions?